docs/visual-basic/misc/bc30665.md

```diff
@@ -24,5 +24,5 @@ The argument supplied to `Throw` must either be an instance of `System.Exception
 ## See also
 - [Throw Statement](../../visual-basic/language-reference/statements/throw-statement.md)
 - [Try...Catch...Finally Statement](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
-- [Exception Class in Visual Basic](https://msdn.microsoft.com/library/9aac396f-34ca-4afb-8e6c-e523cb690ba9)
-- [Exception and Error Handling in Visual Basic](https://msdn.microsoft.com/library/3e351e73-cf23-40ab-8b60-05794160529e)
+- [Exception Class in Visual Basic](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/5ytxh3s3(v=vs.100))
```

Comment:
I'd change all the links that go to 9aac396f-34ca-4afb-8e6c-e523cb690ba9 to go to System.Exception class instead using xref